WordPress/src/main/res/values/strings.xml

```diff
@@ -2745,9 +2745,13 @@
     <string name="prepublishing_nudges_publish_action">Publish</string>
     <string name="prepublishing_nudges_visibility_action">Visibility</string>
     <string name="prepublishing_nudges_tags_action">Tags</string>
+    <string name="prepublishing_nudges_home_header_publishing_to">PUBLISHING TO</string>
```

Comment:
I'd consider using `android:textAllCaps="true"` in the xml layout. This approach might not work as expected with GlotPress. Wdyt?

Reply:
Good eye 😉 
532bac6